docs/standard/serialization/binaryformatter-migration-guide/winforms-migration-guide.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "BinaryFormatter Migration Guide: WinForms Migration Guide"
+description: "Learn about the effects of the deprecation and removal of BinaryFormatter from .NET on Windows Forms and how to migrate."
+ms.date: 7/31/2024
+no-loc: [BinaryFormatter, WinForms]
+helpviewer_keywords:
+  - "BinaryFormatter"
+  - "WinForms"
+---
+
+# Window Forms migration guide for BinaryFormatter
+
+## BinaryFormatter removal
+
+Starting in .NET 9, `BinaryFormatter` has moved to a separate, unsupported NuGet package due to its known [security risks](../binaryformatter-security-guide.md). For more information about the risks `BinaryFormatter` poses and the reason for its removal, see [BinaryFormatter migration guide](overview.md). With BinaryFormatter’s removal, it's expected that many Windows Forms applications will be impacted, and you'll need to take action to complete your migration to .NET 9 or a later version.
```

Comment:
Should BinaryFormatter word be emphasized with tick marks everywhere in the doc?

Reply:
I'm unsure if we should backtick BinaryFormatter everywhere. It seems like a distraction in most places and leaving the backticks off helps solidify that BinaryFormatter is as much a _concept_ as it is n _API_. I'm inclined to only backtick it when specifically referring to the type name, but omit the backticks when referring to the _concept_.

What do you think, @gewarren?

Reply:
I tend to agree with @jeffhandley. Even without backticks, BinaryFormatter is unlikely to get localized.